src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -12534,8 +12540,8 @@ INSERT INTO tab_container_parameters VALUES(4701,'GENERAL','MoleculeProperties',
 INSERT INTO tab_container_parameters VALUES(4711,'GENERAL','MoleculeProperties','Sum of active process rates lumen to mucosa',0,1,0,0,'UNDEFINED',1,'LOCAL','SIMULATION',0,0);
 INSERT INTO tab_container_parameters VALUES(4714,'GENERAL','MoleculeProperties','Sum of active process rates lumen to mucosa',0,1,0,0,'UNDEFINED',1,'LOCAL','SIMULATION',0,0);
 INSERT INTO tab_container_parameters VALUES(157,'ORGAN','Kidney','eGFR',0,1,1,0,'GFR',99,'LOCAL','INDIVIDUAL',0,1);
-INSERT INTO tab_container_parameters VALUES(5015,'GENERAL','PROTEIN','Disease factor',1,0,1,0,'DISEASE_STATES',1,'GLOBAL','INDIVIDUAL',0,0);
-INSERT INTO tab_container_parameters VALUES(5941,'GENERAL','TRANSPORTER','Disease factor',1,0,1,0,'DISEASE_STATES',1,'GLOBAL','INDIVIDUAL',0,0);
+INSERT INTO tab_container_parameters VALUES(5015,'GENERAL','PROTEIN','Disease factor',1,0,1,0,'DISEASE_STATE_FACTORS',1,'GLOBAL','INDIVIDUAL',0,0);
+INSERT INTO tab_container_parameters VALUES(5941,'GENERAL','TRANSPORTER','Disease factor',1,0,1,0,'DISEASE_STATE_FACTORS',1,'GLOBAL','INDIVIDUAL',0,0);

```

Comment:
I also created this group here so that those factors are only visible in advanced mode

Reply:
![image](https://user-images.githubusercontent.com/1041237/159557168-9456972b-242a-452f-a84b-ba712ff075ef.png)

Reply:
@msevestre The PR looks good for me!

Has nothing to do with the PR, but where you are changing the DB right now: can you change **IN DEVELOPMENT** to **ß version** in the description of Cat & Cattle? E.g. `Cat (ß version)`
![grafik](https://user-images.githubusercontent.com/25061876/159674499-df4729f3-63f3-4f1a-a11f-7c5b4a6c8030.png)